### PR TITLE
Add asset limit + tests

### PR DESCRIPTION
**Pull request by nikhilwoodruff**
# SNAP asset limit logic + test

- [x] Label field added
- [x] Documentation field added
- [x] Unit field added
- [x] Default value field added if relevant
- [x] Variable name follows conventions
- [x] Unit test(s) added
- [x] Integration test(s) added if relevant
- [ ] Issues this PR fixes linked

## What's changed

Added the federal SNAP limit parameters (limits with and without elderly/disabled members) for 2021 and 22.

### CODE REVIEW

File: openfisca_us/parameters/usda/snap/asset_test/limit.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+          href: https://fns-prod.azureedge.net/sites/default/files/resource-files/COLAMemoFY2021.pdf
+    2021-10-01: 
+      value: 2_500
+      reference:
+        - title: SNAP - Fiscal Year 2022 Cost-of-Living Adjustments
+          href: https://fns-prod.azureedge.net/sites/default/files/resource-files/2022-SNAP-COLA-%20Maximum-Allotments.pdf
+elderly_disabled:
+  description: Maximum value of SNAP-visible assets for households with elderly or disabled members
+  values:
+    2010-10-01:
```
Comment (MaxGhenis) [inline openfisca_us/parameters/usda/snap/asset_test/limit.yaml:18]: At least per the source
```suggestion
    2020-10-01:
```

File: openfisca_us/parameters/usda/snap/asset_test/limit.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+      value: 3_500
+      reference:
+        - title: SNAP - Fiscal Year 2021 Cost-of-Living Adjustments
+          href: https://fns-prod.azureedge.net/sites/default/files/resource-files/COLAMemoFY2021.pdf
```
Comment (MaxGhenis) [inline openfisca_us/parameters/usda/snap/asset_test/limit.yaml:22]:
```suggestion
          href: https://fns-prod.azureedge.net/sites/default/files/resource-files/COLAMemoFY2021.pdf#page=5
```

File: openfisca_us/parameters/usda/snap/asset_test/limit.yaml [hunk 3]
```diff
@@ -0,0 +1,32 @@
+description: Maximum value of SNAP-visible assets
+standard:
+  description: Maximum value of SNAP-visible assets for households without elderly or disabled members
+  values:
+    2020-10-01:
+      value: 2_250
+      reference:
+        - title: SNAP - Fiscal Year 2021 Cost-of-Living Adjustments
+          href: https://fns-prod.azureedge.net/sites/default/files/resource-files/COLAMemoFY2021.pdf
```
Comment (MaxGhenis) [inline openfisca_us/parameters/usda/snap/asset_test/limit.yaml:9]:
```suggestion
          href: https://fns-prod.azureedge.net/sites/default/files/resource-files/COLAMemoFY2021.pdf#page=5
```

File: openfisca_us/parameters/usda/snap/asset_test/limit.yaml [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+    2021-10-01: 
+      value: 2_500
+      reference:
+        - title: SNAP - Fiscal Year 2022 Cost-of-Living Adjustments
+          href: https://fns-prod.azureedge.net/sites/default/files/resource-files/2022-SNAP-COLA-%20Maximum-Allotments.pdf
```
Comment (MaxGhenis) [inline openfisca_us/parameters/usda/snap/asset_test/limit.yaml:14]: I don't see asset limits here

File: openfisca_us/tests/policy/baseline/usda/snap/asset_test.yaml [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+  output:
+    meets_snap_asset_test: true
+- name: Assets over limit for an elderly/disabled household
+  period: 2022
+  absolute_error_margin: 0
+  input:
+    snap_assets: 3_800
+    has_elderly_disabled: true
+  output:
+    meets_snap_asset_test: false
```
Comment (MaxGhenis) [inline openfisca_us/tests/policy/baseline/usda/snap/asset_test.yaml:30]:
```suggestion
    meets_snap_asset_test: false

```

**Review by MaxGhenis — APPROVED** [review]
LGTM, will we have a separate state-level parameter to override this? Only 14 states currently have SNAP asset tests.

File: openfisca_us/parameters/usda/snap/asset_test/limit.yaml (hunk 4 above)
Comment (nikhilwoodruff) [reply openfisca_us/parameters/usda/snap/asset_test/limit.yaml]: Link corrected

**Comment by nikhilwoodruff** [comment]
> LGTM, will we have a separate state-level parameter to override this? Only 14 states currently have SNAP asset tests.

Yes, I'll add the categorical eligibility parameters and logic separately.

**Pull request merged**